modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,275 +64,456 @@ and device ID 0x1015
-          hostPath:
-            path: /var/lib/cni/bin
-----
-
-. Run the following command to create the `sriov-cni` DaemonSet:
-+
-----
-$ oc create -f sriov-cni.yaml
+  dpConfigVersion: d41d8cd98f00b204e9800998ecf8427e
+status:
+  interfaces:
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f0
+    pciAddress: "0000:18:00.0"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f1
+    pciAddress: "0000:18:00.1"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f0
+    pciAddress: 0000:81:00.0
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f1
+    pciAddress: 0000:81:00.1
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens803f0
+    pciAddress: 0000:86:00.0
+    totalvfs: 64
+    vendor: "8086"
+  syncStatus: Succeeded
 ----
 
-== Configuring additional interfaces using SR-IOV
+== Configuring SR-IOV network devices
+
+The SR-IOV Network Operator introduces the `SriovNetworkNodePolicy` Custom Resource
+Definition (CRD) to specify the SR-IOV network device and the configuration of
+SR-IOV device plug-in.
+
+You should never have to modify this CRD. To make changes to your deployment,
+create and modify a specific Custom Resource (CR). Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+To make the configuration change take effect, creating or modifying the Custom
+Resource of `SriovNetworkNodePolicy` may trigger operator to drain the nodes,
+and in some cases reboot the nodes.
+
+The whole process may take several minutes. All configuration changes shall
+have been applied until all the Pods in `sriov-network-operator` namespace are
+in `Running` status.
+=====
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodePolicy`.
 
-. Create a YAML file for the Custom Resource (CR) with SR-IOV configuration. The
-`name` field in the following CR  has the value `sriov-conf`.
-+
 [source,yaml]
 ----
-apiVersion: "k8s.cni.cncf.io/v1"
-kind: NetworkAttachmentDefinition
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
 metadata:
-  name: sriov-conf
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: openshift.io/sriov <1>
+  name: policy-example <1>
+  namespace: sriov-network-operator <2>
 spec:
-  config: '{
-      "type": "sriov", <2>
-      "name": "sriov-conf",
-      "ipam": {
-        "type": "host-local",
-        "subnet": "10.56.217.0/24",
-        "routes": [{
-          "dst": "0.0.0.0/0"
-        }],
-        "gateway": "10.56.217.1"
-      }
-    }'
+  resourceName: sriov <3>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true" <4>
+  priority: 99 <5>
+  mtu: 9000 <6>
+  numVfs: 16 <7>
+  nicSelector: <8>
+    vendor: "15b3" <8>
+    deviceID: "" <9>
```

Comment:
Starting from `<9>`, the explanations below don't match with options.

Reply:
Off-by-one. Updated.